platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functionnal/SOCNotificationIntranetNotificationIconTestIT.java

```diff
@@ -0,0 +1,207 @@
+import static org.exoplatform.platform.qa.ui.selenium.locator.NavigationToolBarLocator.ELEMENT_BADGE_NUMBER_DISPLAY;
+import static org.exoplatform.platform.qa.ui.selenium.locator.NavigationToolBarLocator.ELEMENT_INTRANET_NOTIFICATION_NEAR_USER_AVATAR;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+@Tag("social")
+@Tag("sniff")
+public class SOCNotificationIntranetNotificationIconTestIT extends Base {
+    NavigationToolbar navigationToolbar;
+    ManageLogInOut manageLogInOut;
+    AddUsers addUsers;
+    ConnectionsManagement connectionsManagement;
+    HomePagePlatform homePagePlatform;
+    IntranetNotification intranetNotification;
+    MyNotificationsSetting MyNotificationsSetting;
+    UserAddManagement UserAddManagement;
+    ArrayList<String> arrayUser;
+    UserProfilePage userProfilePage;
+    UserAndGroupManagement UserAndGroupManagement;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+        navigationToolbar = new NavigationToolbar(this);
+        manageLogInOut = new ManageLogInOut(this);
+        addUsers = new AddUsers(this);
+        connectionsManagement = new ConnectionsManagement(this);
+        homePagePlatform = new HomePagePlatform(this);
+        manageLogInOut.signInCas(USER_ROOT, DATA_PASS);
+        MyNotificationsSetting = new MyNotificationsSetting(this);
+        UserAddManagement = new UserAddManagement(this);
+        userProfilePage = new UserProfilePage(this);
+        intranetNotification = new IntranetNotification(this);
+        UserAndGroupManagement = new UserAndGroupManagement(this);
+    }
+
+    /**
+     * <li> Case ID:125112.</li>
+     * <li> Test Case Name: Check Notifications icon in the top navigation.</li>
+     * <li> Pre-Condition: </li>
+     * <li> Post-Condition: </li>
+     */
+    /*Step Number: 1
+		*Step Name:
+		*Step Description:
+			- Login to platform
+		*Input Data:
+
+		*Expected Outcome:
+			- The homepage is displayed*/
+
+		/*Step number: 2
+		*Step Name:
+		*Step Description:
+			- Check the top navigation
+		*Input Data:
+
+		*Expected Outcome:
+			- The Notifications icon is displayed in the top navigation, next to the profile menu*/
+    @Test
+    public void test01_CheckNotificationsIconInTheTopNavigation() {
+        info("Test 1: Check Notifications icon in the top navigation");
+        String username1 = "usernamea" + getRandomString();
+        String email1 = username1 + "@gmail.com";
+        String password = "123456";
+
+        info("Add new user");
+        navigationToolbar.goToAddUser();
+        UserAddManagement.addUser(username1, "123456", email1, username1, username1);
+        manageLogInOut.signIn(username1, "123456");
+
+        info("Check The Notifications icon is displayed in the top navigation");
+        $(ELEMENT_INTRANET_NOTIFICATION_NEAR_USER_AVATAR).waitUntil(Condition.visible, Configuration.timeout);
```

Comment:
you must delete data at the end of test (same for other tests)

Reply:
done